### PR TITLE
EVG-2427 add some indices

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: scripts/indexes.js [hunk 1]
```diff
@@ -103,3 +104,9 @@ db.json.ensureIndex({ "version_id" : 1 })
 
 //======testresults======//
 db.testresults.ensureIndex({ "task_id" : 1, "task_execution" : 1 })
+
+//======project_aliases======//
+db.testresults.ensureIndex({ "project_id" : 1, "alias" : 1 })
```
Comment (bsamek) [inline scripts/indexes.js]: db.project_aliases

File: scripts/indexes.js [hunk 2]
```diff
@@ -103,3 +104,9 @@ db.json.ensureIndex({ "version_id" : 1 })
...
+
+//======patch_intents======//
+db.testresults.ensureIndex({ "processed" : 1, "intent_type" : 1 })
```
Comment (bsamek) [inline scripts/indexes.js]: db.patch_intents

File: scripts/indexes.js [hunk 3]
```diff
@@ -103,3 +104,9 @@ db.json.ensureIndex({ "version_id" : 1 })
 
 //======testresults======//
 db.testresults.ensureIndex({ "task_id" : 1, "task_execution" : 1 })
+
+//======project_aliases======//
+db.testresults.ensureIndex({ "project_id" : 1, "alias" : 1 })
+
+//======patch_intents======//
```
Comment (bsamek) [inline scripts/indexes.js]: remove line 45

Comment (john-m-liu) [reply scripts/indexes.js]: done

File: scripts/indexes.js (hunk 2 above)
Comment (john-m-liu) [reply scripts/indexes.js]: done

File: scripts/indexes.js (hunk 1 above)
Comment (john-m-liu) [reply scripts/indexes.js]: done

**Pull request merged**